client/client.go

```diff
@@ -337,6 +343,12 @@ type SecurityOption struct {
 	SSLKEYBytes  []byte
 }
 
+// KeyRange represents a range of keys.
+type KeyRange struct {
```

Comment:
We have the same definition in `client/http` pkg, is it possible to reuse it?

https://github.com/tikv/pd/blob/c1d422ec4924799a5ff7da6606644959e339f1c0/client/http/types.go#L47

Reply:
Can I simply made the `startKey` and `endKey` of this struct public? If so, we can certainly reuse it.

Reply:
I moved the `KeyRange` struct from `client/http` pkg to `client` pkg(unless there will be cycling imports).
To avoid breaking compatibility, an alias is made in `client/http` pkg.